src/index.ts

```diff
@@ -94,38 +95,38 @@ const config = Config.fromFile("./config.toml");
 
   const shoot = async () => {
     const swaps = await comitNode.getSwaps();
-    log(`Found ${swaps.length} swap(s)`);
+    logger.trace(`Found ${swaps.length} swap(s)`);
 
     for (const swap of swaps) {
       const id = swap.properties ? swap.properties.id : undefined;
       try {
         const selectedAction = await actionSelector.selectActions(swap);
         if (selectedAction) {
-          log(
+          logger.trace(
             `Selected action for swap ${id}: ${JSON.stringify(selectedAction)}`
           );
           const executionResult = await actionExecutor.execute(
             selectedAction,
             config.maxRetries
           );
-          log(
+          logger.trace(
             `Action execution response for swap ${id}: ${JSON.stringify(
               executionResult
             )}`
           );
         } else {
-          log(`No action returned for swap ${id}`);
+          logger.trace(`No action returned for swap ${id}`);
         }
       } catch (err) {
-        log(
+        logger.debug(
           `Error has occurred for swap ${id}. Error is: ${JSON.stringify(err)}`
         );
       }
     }
   };
 
   setInterval(() => {
-    shoot().then(() => log("Execution done"));
+    shoot().then(() => logger.info("Execution done"));
```

Comment:
📣 I don't think this log makes any sense. Execution of what? Can you investigate a bit and see we can make it meaningful?